README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Rozdział I – Postanowienia ogólne
 
-§ 1. Stowarzyszenie nosi nazwę „Stowarzyszenie Hakierspejs Łódź”, w dalszych postanowieniach zwane Stowarzyszeniem.
+§ 1. Stowarzyszenie, w dalszych postanowieniach zwane Stowarzyszeniem, nosi nazwę „Stowarzyszenie Hakierspejs Łódź”.
```

Comment:
nie zmieniałbym tego, bo mimo że nie ma to konsekwencji, to wpływa na tekst jednolity i komplikuje spisanie protokołu

Reply:
Czyli co, mamy mieć nieścisłość między tym, co już poszło (parę miesięcy temu) do sądu, a tym co jest na GH? Bez sensu, to trzeba wyrównać.

Reply:
A, w tę stronę? To spoko, niech idzie xD

Reply:
Potwierdzam: sprawdziłem treść wydrukowanego statutu i jest tam ten punkt w wersji takiej, jak na zielono.